web/src/components/BottomActionSheet.tsx

```diff
@@ -56,7 +56,7 @@ const BottomActionSheet = React.forwardRef(
         open
         scrollLocking={false}
         blocking={false}
-        header={!!title && <Title>{title}</Title>}
+        header={!title ? <Title>{title}</Title> : null}
```

Comment:
I think this should be
```suggestion
        header={title ? <Title>{title}</Title> : null}
```